server/src/services/NotesService.ts

```diff
@@ -0,0 +1,15 @@
+import { Service } from '@tsed/common'
+import { MongooseService } from '@tsed/mongoose'
+
+// To retrieve instance of Mongoose.Connection
+
+@Service()
+export class NotesService {
```

Comment:
I think it is not necessary to add service if you planning to have only default connection